### DISCUSSION TITLE
Permanent free users

### DISCUSSION THREAD

**Discussion by d9vsky** [Glench/ExtPay#202] [Q&A] [answered]
Hi, 
I am wondering how does the permanent free users functionality works. Basically, we are able only to go there through "Add free users" link. Then we get below window:
![image](https://github.com/Glench/ExtPay/assets/130619620/d4583c48-b67f-49aa-9bc9-0b62a7741e54)

Where the emails section is always empty - giving us possibility to add free users in a bulk. 
But how can we control a list of free users? How to clear it?
What is the process of adding free users? Do I always need to input a complete list of free users and extend them with new ones?
Is there any chance that there will be a dashboard of free users, which will give us the possibility to manipulate it easily?

Best Regards,
Mateusz

**Comment by d9vsky** [comment]
@Glench ?

**Reply by Glench** [reply, marked as answer]
As mentioned in another thread, please just email us (glen@extensionpay.com) for now if you want to see or remove your list of free users. We'll update this at some point, but right now we're focused on getting another feature out.

Update: you can now just manage your permanent free users yourself.

**Reply by salcido** [reply]
@Glench Can you explain how to add / manage free users? I need to provide a test account for Chrome / Firefox reviewers and I don't see an obvious way to do this.

**Reply by burkybang** [reply]
@salcido

1. Go to https://extensionpay.com/home
2. Click "Add free users"

<img width="446" height="498" alt="image" src="https://github.com/user-attachments/assets/fc18d974-3758-4979-917a-5e3165bfb7df" />

**Reply by salcido** [reply]
@burkybang Thanks! Totally forgot about this page. I've been spending all my time in Stripe and thought I had to do it through their UI.
